astacus/coordinator/plugins/files.py

```diff
@@ -15,6 +15,7 @@
 
 """
 
+from ..cluster import Cluster
```

Comment:
Please use absolute path in non-local imports (this is in few other places too, but just noting it where I first noticed it)